examples/widgets/assets/world.meta

```diff
@@ -0,0 +1,5 @@
+assets/1.0x/world.png
```

Comment:
We shouldn't check the material icons into this repo. I recommend making some random trivial artwork and then scaling it down. Maybe ask the UX guys for something. :-)